### DISCUSSION TITLE
Test Replay does not display all the information from cypress-plugin-api

### DISCUSSION THREAD

**Discussion by juan-belmonte** [cypress-io/cypress#27796] [Cypress Cloud]
After successfully upgrading to Cypress 13, we ran our API tests built using [`cypress-plugin-api`](https://github.com/filiphric/cypress-plugin-api) to check if Test Replay was suitable for this kind of tests &mdash; actual [requests are made in Node](https://docs.cypress.io/api/commands/request#Request-is-not-displayed-in-the-Network-Tab-of-Developer-Tools), not in the browser.

As far as I can extract from the output, only the _number_ of requests, the response status code and duration are successfully displayed on screen. The debugging information of the requests &mdash; URL, headers, body from both the request and the response &mdash; are blank. The `cypress-plugin-api` UI is shown, though.

<img width="1569" alt="cypress-plugin-api-test-replay" src="https://github.com/cypress-io/cypress/assets/88533684/2e19fe86-93a8-4628-87fb-f3de511b59d8">

Is Test Replay expected to successfully display all the information from `cy.request` and thus `cy.api`? Is `cy.request` outside its scope? Or is this an integration issue between the new Test Replay and the plugin itself?

**Comment by ztarsoly** [comment]
I can confirm that cy.request command has no any output in the network tab (which is okay) or in the console tab when I click on the command (which is not okay).

**Comment by juan-belmonte** [comment]
We get no console output when clicking on the step that's failing:
```
- assert expected { Object (code, data, ...) } to deeply match { Object (code, data, ...) }
```

We are still blind to the root cause for these API tests, unless we repeat it locally for sure.
That's why I still wonder if _this an integration issue between the new Test Replay and the plugin itself_.

**Comment by jennifer-shehane** [comment]
We have plans to better support `cy.request` in Test Replay - we recognize the review of this is not a great experience outside of the App. Today you should be able to click on the actual `cy.request` command with your devtools open to see console information about it.

**Reply by juan-belmonte** [reply]
Thank you for your reply @jennifer-shehane. I'll keep an eye on the communications so that I'm aware when support for `cy.request` is enhanced.
